lessonbuilder/tool/src/webapp/js/lessons-calendar.js

```diff
@@ -2,6 +2,8 @@ $(function(){
     let url = $('.site-events-url').first().text().replace(/'/g,'');
     let moreInfoUrl = $('.event-tool-url').text().replace(/'/g,'');
 
+    const userLanguage = window.top.portal.locale || navigator.language;
```

Comment:
i'm not sure i agree with using the navigator.language at all here. sakai has always used the portal preference. so if the rest of sakai doesn't use navigator.language, you shouldn't use it here

Reply:
`calendar/sakai-calendar.js` is using navigtor : https://github.com/sakaiproject/sakai/blob/5efb25acb549b55831d4608b440d0038bf372ca1/calendar/calendar-tool/tool/src/webapp/js/sakai-calendar.js#L3

Reply:
looks wrong too? 

sakai allows:

1) the user to set their preference
2) The site to have a preference
3) the entire instance to have a preference